admin-panel/src/main/java/net/cryptic_game/backend/admin/Config.java

```diff
@@ -0,0 +1,17 @@
+package net.cryptic_game.backend.admin;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Getter
+@Component
+public class Config {
+
+    @Value("${API_TOKEN:}")
+    private String apiToken;
+
+    @Value("${SERVER_URL:}")
+    private String serverUrl;
+
+}
```

Comment:
Please use Spring Boot custom properties. 
https://www.baeldung.com/configuration-properties-in-spring-boot#1-spring-boot-22